ext/standard/string.c

```diff
@@ -1866,6 +1866,44 @@ PHP_FUNCTION(str_contains)
 }
 /* }}} */
 
+/* {{{ proto boolean str_starts_with(string haystack, string needle)
```

Comment:
s/boolean/bool/ (like everywhere else in the file)
Same for str_ends_with below